.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Signaler un bug
+about: Je viens de rencontrer un problème.
+title: '[BUG]'
+labels: bug
+assignees: HAEKADI
+---
+
+## Sujet du problème
+
+Décrivez votre problème ici.
+
+## Comment le reproduire ?
+
+Étapes à suivre pour faire apparaître le bug.
```

Comment:
I think it might be a good idea to add this to the website.

```suggestion
Étapes à suivre pour faire apparaître le bug :
1. Aller à `…`
2. Cliquer sur `…`
3. Naviguer jusqu’à `…`
4. Constater le bug
```